tests/Formatting/Formatters/FullyQualifiedFacadesTest.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Tests\Formatting\Formatters;
+
+use PHPUnit\Framework\TestCase;
+use Tighten\TLint\Formatters\FullyQualifiedFacades;
+use Tighten\TLint\TFormat;
+
+class FullyQualifiedFacadesTest extends TestCase
+{
+    /** @test */
+    public function fixes_facade_aliases()
+    {
+        $file = <<<'file'
+<?php
+
+namespace Test;
+
+use DB;
+use Storage;
+
+return true;
+file;
+
+        $formatted = (new TFormat)->format(new FullyQualifiedFacades($file));
+
+        $expected = <<<'file'
+<?php
+
+namespace Test;
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Storage;
+
+return true;
+file;
+
+        $this->assertSame($expected, $formatted);
+    }
+
+    /** @test */
+    public function fixes_facade_aliases_when_file_not_namespaced()
+    {
+        $file = <<<'file'
+<?php
+
+use DB;
+use Storage;
+
+return true;
+file;
+
+        $formatted = (new TFormat)->format(new FullyQualifiedFacades($file));
+
+        $expected = <<<'file'
+<?php
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Storage;
+
+return true;
+file;
+
+        $this->assertSame($expected, $formatted);
+    }
+
+    /** @test */
+    public function fixes_facade_aliases_mixed_with_valid_use_statements()
```

Comment:
Could you also run a test where it's not the first entry?

Reply:
This test also has `use Storage;` => `use Illuminate\Support\Facades\Storage;`